models/perm/access/repo_permission.go

```diff
@@ -25,7 +25,8 @@ type Permission struct {
 	units     []*repo_model.RepoUnit
 	unitsMode map[unit.Type]perm_model.AccessMode
 
-	everyoneAccessMode map[unit.Type]perm_model.AccessMode
+	everyoneAccessMode  map[unit.Type]perm_model.AccessMode
+	anonymousAccessMode map[unit.Type]perm_model.AccessMode
```

Comment:
Could we have some comments here about `everyoneAccessMode` and `anonymousAccessMode`?

Reply:
Added in https://github.com/go-gitea/gitea/pull/33257/commits/819485e1c9c8b4b0feb88893f49af6769b98c5d3